examples/basic-tracer/src/main.rs

```diff
@@ -0,0 +1,18 @@
+use opentelemetry::sdk::export::trace::stdout;
+use opentelemetry::sdk::trace::{self, Sampler};
+use opentelemetry::{trace::Tracer, KeyValue};
+
+fn init_tracer() -> impl Tracer {
+    let v = vec![KeyValue::new("key", "value")];
+    stdout::new_pipeline()
+        .with_trace_config(trace::config().with_sampler(Sampler::AlwaysOn))
+        .install_with_tracer_attributes(v)
+}
+
+#[tokio::main]
```

Comment:
nit: if this is truly intending to show a most basic tracing example, we don't need async, tokio here, right?

Reply:
Removed.